src/legacy/core_plugins/kibana/ui_setting_defaults.js

```diff
@@ -426,8 +426,9 @@ export function getUiSettingDefaults() {
       type: 'boolean',
       description: i18n.translate('kbn.advancedSettings.courier.batchSearchesText', {
         defaultMessage:
-          'Batch multiple concurrent searches in a single request. This may improve overall load' +
-          'times, but it also means that no single search request will be returned until the last has completed.',
+          `When disabled, dashboard panels will load individually, and search requests will be terminated when users
+           navigate away or update the query. When enabled, dashboard panels will load altogether when all of the
+           data has been loaded.`,
       }),
```

Comment:
I like this description.  When I looked at the UI, I noticed that the switch is on by default.  So how about we describe the default behavior and then explain what happens when you disable:

<img width="590" alt="Screen Shot 2019-09-25 at 12 52 08 PM" src="https://user-images.githubusercontent.com/33642766/65634695-ab61a300-df93-11e9-8519-912adb921a82.png">

What do you think?